.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -96,7 +96,7 @@ jobs:
           apt-get install postgresql-${PG_MAJOR}-dbgsym >/dev/null
           for file in /tmp/core*
           do
-            echo 'printf "%s", debug_query_string\nbt full' | gdb /usr/lib/postgresql/${PG_MAJOR}/bin/postgres -c $file | tee -a stacktraces.log
+            echo -e 'printf "%s", debug_query_string\nbt full' | gdb /usr/lib/postgresql/${PG_MAJOR}/bin/postgres -c $file | tee -a stacktraces.log
```

Comment:
This is dependent on the implementation of `echo`, which should be fairly stable but always a potential risk. I prefer to use here-is for this situation since it always works.

```bash
gdb /usr/lib/postgresql/${PG_MAJOR}/bin/postgres -c $file <<EOT | tee -a stacktraces.log
printf "%s", debug_query_string
bt full
EOT
```

Reply:
@mkindahl That's a good point, and I think it's easier to read using the heredoc syntax. Unfortunately, the delimiter is not allowed to be indented (or only tabbed), so it is at the beginning of the line in the updated PR.